tomls/permit-deniers.toml

```diff
@@ -13,7 +13,7 @@ args = [
         "0x599e835cbFC903eF09f3Dd5E08D1cF63c32AF8d8",
         "0x347c3190bD015FBD0e47fb90AA4917138A8A32FE",
         "0x562948111d50BF039A39Eea48D127f2Ae51ddF02",
-        "0x1C8236B406911A376369e33D39189F1b4B39F27D",
+        "<%= settings.deployer %>",
```

Comment:
should we consider not having a depolyer as a denier at all maybe?

Reply:
its been a discussion before. not sure if now is a good time to be rethinking this.

Reply:
considering how few pdao signers we have and the minimal concern if someone were to deny the system, doesn't seem too important to remove at this time.